frontend/src/components/query/overwrite-sql-dialog.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+import { Pipeline } from "@/definitions";
+import {
+  Button,
+  Callout,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Text,
+} from "@blueprintjs/core";
+import { useUpdateUserQuery } from "@/data/use-user-query";
+import Loading from "@/app/loading";
+
+interface OverwriteSQLDialogProps {
+  userQueryId: string;
+  pipeline: Pipeline;
+  divergence: boolean;
```

Comment:
I think this needs a comment or similar, divergence is the difference between the two? do we need to know what the divergence is, or just that it IS divergent?

If simply the boolean, we should always use our convention: isX, and always in the "affirmative"

in this case, `isDivergent`